charts/tigera-operator/templates/crs/custom-resources.yaml

```diff
@@ -27,6 +27,13 @@ spec:
 {{ if .Values.whisker.enabled }}
 ---
 
+apiVersion: operator.tigera.io/v1
```

Comment:
We need to follow the example of the other CRs here and provide a mechanism to enable / disable via the helm chart values.yaml

Reply:
OK, I was thinking we could tie this to the whisker flag, but I'm fine to add a different flag.

Reply:
One point of having them as separate CRs is that it's possible to deploy Goldmane without Whisker, so we probably want to show that here.